out.py

```diff
@@ -15,22 +15,27 @@ class _CursorInfo(ctypes.Structure):
 
 
 def error(message: str, context: Any = __UNIT) -> None:
-    print(message, file=sys.stderr)
+    println(danger(message), file=sys.stderr)
     if context is not __UNIT:
-        print(context, file=sys.stderr)
+        println(context, file=sys.stderr)
 
 
 RESET = "\033[0m"
-PRIMARY = "\033[95m"
+PRIMARY = "\033[92m"
 SECONDARY = "\033[94m"
+DANGER = "\033[91m"
 
 
-def println(*args, end: str = "\n") -> None:
-    sys.stdout.write(" ".join(map(str, args)) + end)
+def println(*args, end: str = "\n", file: TextIO = sys.stdout) -> None:
+    print(*args, end=end, file=file)
 
 
-def flush() -> None:
-    sys.stdout.flush()
+def flush(file: TextIO = None) -> None:
```

Comment:
inconsistent behavior with out.println

Reply:
what do you mean ?

Reply:
println with no arguments prints only to stdout, therefore flush with no arguments should flush only stdout